### ISSUE TITLE
Make setup instructions more inclusive

### ISSUE THREAD

**Issue by mturoci** [h2oai/wave-apps#108]
Current setup instructions require `make` to be installed. However, this can be a showstopper for windows users which lacks `make` installation by default.

We could either:

* Rewrite the instructions to plain python commands.
* Find OS agnostic alternative to `make`.

cc @mtanco @geomodular for discussion

**Comment by geomodular** [comment]
Would be great to find a typical user for this kind of use case and discuss it. From my perspective, I suggest to:
- keep `Makefile`,
- write a setup guide (this would be effectively the commands from `Makefile`) so the user can run commands by hand without `Makefile`,
- create `.bat` files (if it makes sense) - I assume this is an alternative to scripts in Windows for a developer (sorry for my incompetence),
- update README of several ways how to run the project (`Makefile`, by hand, `.bat`)

> Rewrite the instructions to plain python commands.

I'd refrain from using Python as a build/setup tool. I think it's not necessary here.

**Comment by denzuko** [comment]
Yes, make is rather ubiquitous in POSIX and absent (by design)™ in Microsoft. But what I have found over the years is that [PyInvoke](https://www.pyinvoke.org) is the closes thing to an agnostic replacement for Make. Mind you that means one has to rewrite thier makefiles into a `tasks.py` file but it is a near one to one since each @task decorated function's `context` parameter has a method called `.run()` which calls a shell just like make does while the dependency of tasks is just a list passed to the `pre` parameter to the @task decorator.

That said, if one still wants to use Makefiles on non-POSIX systems then there is Cygwin, MSYS, Docker, and the newer WSL2. which ports a POSIX environment into Win64 systems.

My vote is for PyInvoke seeing how the ecosystem is mainly python based and developers can grok python easier than batch files or Makefiles.
